### PR TITLE
Salute

### LINKED ISSUE #81: Add salute emoji (go crazyyyyy xd) and clean up styling routes
- [x] Add salute emoji anywhere u want LOLLLLL
- [x] I noticed we still have some component files that still import from the email sender style file or other page styles files. The components should only import from their respective style file (if the styled component is not used elsewhere) or imported from the common file if they are shared. Styled components should only be imported from the email sender style file if they are used on emailSender.tsx (same as for the other pages) 
- [x] Fix all the style imports / where the styled components live accordingly 
Screen Shot 2022-08-06 at 4 33 07 PM

### PR DESCRIPTION
**Pull request by Dhivas-Sugumar**
Added salute and fixed style imports. 

Fixes #81.

Changelist:
- Added salute (most important change)!!!!
- Also fixed imports for styles. 

Tested:
- No tests are necessary. 

p.s find the salutes without using the code!!!

### CODE REVIEW

File: components/displayMessages/displayMessages.tsx
Comment (jzhzhang66) [inline components/displayMessages/displayMessages.tsx:3]: Can we move this into common also?

**Review by jzhzhang66 — APPROVED** [review]
AYYYYYY the salute is innnnn!!!!!

Approving with one minor comment before merging:)

**Review by karynayen — APPROVED** [review]
YAYAYAY SALUTEEE!!!!
aside from Judy's suggestions, everything LGTM!!!!

**Pull request merged**
